### DISCUSSION TITLE
Response is not displayed in tc report

### DISCUSSION THREAD

**Discussion by erwinmoretz** [thunderclient/thunder-client-support#1307] [Q&A] [answered]
Hi rangav, 

i've got a problem with the response in the tc report. 
I guess there is a size limit for the response in the display of the requests.

I send a GET request via pipeline job with ThunderClient and get "Response not displayed for this request".

![image](https://github.com/rangav/thunder-client-support/assets/112586589/d374c260-a139-4cef-9f66-c6ce4a4e22b2)

Could help me with this issue?

**Comment by rangav** [comment]
Hi @erwinmoretz 

Yes there is size limit currently to display response. And also the response is displayed for first 10 iterations 

do you what is the size of the response? And iteration number

**Comment by erwinmoretz** [comment]
For this Request the size of the Response is ca. 30000 lines. 

But we have also other requests that have bigger Responses (six-digits).

For this Testcollection there is no iteration. :-)

**Reply by rangav** [reply]
Can you share the size displayed in the report, There is a limit of `100kb size` to display the response.

<img width="361" alt="Screenshot 2023-08-31 at 11 33 05" src="https://github.com/rangav/thunder-client-support/assets/8637550/565df03e-6e8d-40e9-8d2e-f836eaf45ff7">

**Reply by erwinmoretz** [reply]
Yes, the size of the response in this request is 869kb
![image](https://github.com/rangav/thunder-client-support/assets/112586589/8572f189-41ed-4e45-aef6-cd678535189e)

Could you tell us where can we change the limit?

**Reply by rangav** [reply]
Currently, the size is not configurable, Will provide a vscode setting in the next update.

**Reply by rangav** [reply]
Can you please create a feature request for this?

**Reply by erwinmoretz** [reply, marked as answer]
#1312

done :-)

**Reply by rangav** [reply]
Thanks :)
